### PR TITLE
DRAFT: feat: Scaida rebranding

### PR DESCRIPTION
**Pull request by chetanppatil**
- added scaida svg logo in assets
- added two css class in tailwind.css according to figma design
- on pages set white theme and new added css class as property to set proper color
- using whitelabling functionality, updated scaida logo

#### PR

- [] My Pull Request title is descriptive, accurate and follows the
  semantic-release format and guidelines.

#### Code

- [] My code has been well-documented (function documentation, inline comments,
  etc.)

#### Public Documentation Updates

- [] The documentation page has been updated as necessary for any public API
  additions or removals.

#### Tested Environment

- [] OS: 
- [] Node version: 
- [] Browser:
  

[blog]: https://circleci.com/blog/triggering-trusted-ci-jobs-on-untrusted-forks/
[script]: https://github.com/jklukas/git-push-fork-to-upstream-branch

### CODE REVIEW

**Comment by chetanppatil** [comment]
Current State Looks Like:

image

File: extensions/default/src/ViewerLayout/index.tsx [hunk 1]
```diff
@@ -59,7 +59,7 @@ function ViewerLayout({
 
   const getPanelData = id => {
     const { content, entry } = getComponent(id);
-
+    console.log('Pratik Entry', entry);
```
Comment (VarshaMLHealth) [inline extensions/default/src/ViewerLayout/index.tsx:62]: Can you please remove this console.log

File: platform/app/public/config/default.js [hunk 2]
```diff
@@ -32,50 +33,130 @@ window.config = {
...
+      response_type: 'token', // "implicit"
+      scope: 'openid api://d6c86043-8a7c-444c-a0a3-07cfd8080532/user_impersonation', // email profile openid  https://dicom.healthcareapis.azure.com
+      // ~ OPTIONAL
+      post_logout_redirect_uri: '/logout-redirect.html',
+      automaticSilentRenew: true,
+      revokeAccessTokenOnSignout: true,
     },
+  ],
+  dataSources: [
+    // {
```
Comment (VarshaMLHealth) [inline platform/app/public/config/default.js]: Remove unwanted code, instead of commenting it

**Review by VarshaMLHealth — CHANGES_REQUESTED** [review]
added review comments Please check
also you forgot to pull changes from The PR i raised. Please pull those changes too.

**Comment by rajangoswamiTWS** [comment]
@Pratik-TWS, @VarshaMLHealth 
What is the status of this PR? Why is this not committed yet?

**Pull request closed without merging**